src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -0,0 +1,43 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+
+public class CommentDataManager{
```

Comment:
Add a JavaDoc explaining the purpose of the class at the top of the ``` class``` declaration.